### DISCUSSION TITLE
Urdu Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#411] [Language Leaders]
@AalianKhan is a current language leader, and @yousaf465 has said they're interested.

@yousaf465, we're asking new contributors to please submit at least two small PRs to become a language leader :+1:

**Comment by AalianKhan** [comment]
@yousaf465 What do you think about adding Roman Urdu?

**Reply by yousaf465** [reply]
> @yousaf465 What do you think about adding Roman Urdu?

Yes, we can use it in aliases

**Comment by yousaf465** [comment]
> @AalianKhan is a current language leader, and @yousaf465 has said they're interested.
> 
> @yousaf465, we're asking new contributors to please submit at least two small PRs to become a language leader 👍

I am fine with @AalianKhan as leader.

**Comment by synesthesiam** [comment]
@AalianKhan @yousaf465 - would you or others have time to translate the new responses into Urdu?

**Reply by AalianKhan** [reply]
Yup I put some time aside today for this

**Reply by synesthesiam** [reply]
Thanks!

**Comment by synesthesiam** [comment]
@AalianKhan when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Reply by AalianKhan** [reply]
Yup, I did them but I wanted to add a translations for other files too so I will make a one big pull request soon

**Reply by synesthesiam** [reply]
Great, thanks!
